histogram_file_manager/models.py

```diff
@@ -3,6 +3,38 @@
 
 # from django.conf import settings
 
+class HistogramDataFileContents(models.Model):
+    GRANULARITY_UNKNOWN = "unk"
```

Comment:
I _think_ we don't need `UNKNOWN` anymore. Could you try removing it completely?

Reply:
Done.